packages/async-flow/test/_utils.js

```diff
@@ -0,0 +1,17 @@
+import { prepareAsyncFlowTools } from '../src/async-flow.js';
+
+/**
+ * @import {Zone} from '@agoric/base-zone';
+ * @import {PreparationOptions} from '../src/types.js';
+ */
+
+/**
+ * @param {*} t
+ * @param {Zone} zone
+ * @param {PreparationOptions} [opts]
+ */
+export const prepareTestAsyncFlowTools = (t, zone, opts) => {
+  const { panicHandler = e => t.log('Handled panic', e) } = opts || {};
```

Comment:
Shouldn't the default behavior be to `t.fail` the test? I know that's what the unhandled rejection handler ava sets up would do, but I think that's the correct default behavior, and here it'd just be more explicit.

Reply:
Okay, so you're talking about the default `panicHandler`.  I presume you mean that the tests that had unhandled rejections, but expected them, would override the `panicHandler` option?